File: chapter2/exercise.py
# Exercise One

hash = "#"
for i in range(0, 7):
    print(hash)
    hash = hash + "#"


# Exercise two

for i in range(1, 101):
    if i % 3 == 0:
        if i % 5 == 0:
            print("FizzBuzz")
        else:
            print("Fizz")
    elif i % 5 == 0:
        print("Buzz")
    else:
        print(i)

# Exercise three

size = 8
chess_board = ""
for i in range(0, size):
    for j in range(0, size):
        if (i + j) % 2 == 0:
            chess_board = chess_board + " "
        else:
            chess_board = chess_board + "#"
    chess_board = chess_board + "\n"

print(chess_board)

File: chapter2/exercise.js
//Exercise One
let hash = '#';
for (let index = 0; index < 7; index++) {
    console.log(hash);
    hash += '#';
}

//Exercise two
for (let index = 1; index <= 100; index++) {
    if (index % 3 == 0) {
        if(index % 5 == 0){
            console.log('FizzBuzz');
        }else{
            console.log('Fizz');
        }
    } else if(index % 5 == 0){
        console.log('Buzz');
    }else{
        console.log(index);
    }
}

//Exercise three
const size = 8;
let chessBoard = '';
for (let outer = 0; outer < size; outer++) {
    for (let inner = 0; inner < size; inner++) {
       if((inner + outer) % 2 == 0){
           chessBoard += ' ';//for the reverse pattern, change this to chessBoard += '#';
       }else{
           chessBoard += '#';//for the reverse pattern, change this to chessBoard += ' ';
       }
    }
    chessBoard += '\n';
}
console.log(chessBoard);